app/OParl/OParlApi.php

```diff
@@ -70,6 +70,20 @@ public function papers(int $page = null)
         return $this->call('GET', $endpoint, $page);
     }
 
+    public function people(int $page = null)
+    {
+        $endpoint = sprintf('body/%s/%s', $this->bodyId, 'person');
```

Comment:
if person is hardcoded here, why not sue sprintf('body/%s/person', $this->bodyId);

Reply:
this issue also exisists though this file..